bindings/CMakeLists.txt

```diff
@@ -5,16 +5,22 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
 if(pyroot)
-# Machinery necessary to loop in parallel over multiple lists
-# This will be used in the CMakeLists of pythonizations,
-# cppyy, cppyy-backend and CPyCppyy to build and install targets
-# for multiple Python versions (only 2 and 3 for now)
-list(LENGTH python_executables len)
-math(EXPR how_many_pythons "${len} - 1")
-
   if(NOT pyroot_experimental)
     add_subdirectory(pyroot)
   else()
+    # Machinery necessary to loop in parallel over multiple lists
+    # This will be used in the CMakeLists of pythonizations,
+    # cppyy, cppyy-backend and CPyCppyy to build and install targets
+    # for multiple Python versions (only 2 and 3 for now)
+    list(LENGTH python_executables len)
+    math(EXPR how_many_pythons "${len} - 1")
+
+    if(IS_ABSOLUTE ${runtimedir})
```

Comment:
this is moved here because now it's needed also for installing libJupyROOT right?

Reply:
Yes exactly, I needed to move it one level up.